x-pack/plugins/alerting/server/rules_client/rules_client.ts

```diff
@@ -875,6 +875,98 @@ export class RulesClient {
     }
   }
 
+  public async getGlobalExecutionLogWithAuth({
+    dateStart,
+    dateEnd,
+    filter,
+    page,
+    perPage,
+    sort,
+  }: GetExecutionLogByIdParams): Promise<IExecutionLogResult> {
+    this.logger.debug(`getExecutionLogForRule(): getting global execution log`);
+
+    let authorizationTuple;
+    try {
+      authorizationTuple = await this.authorization.getFindAuthorizationFilter(
+        AlertingAuthorizationEntity.Alert,
+        alertingAuthorizationFilterOpts
```

Comment:
To get a find authorization filter with fields specific to the event log, you can use different options here:

```
{
  type: AlertingAuthorizationFilterType.KQL,
  fieldNames: { ruleTypeId: 'rule_type_id', consumer: 'kibana.alert.rule.consumer' },
}
```